cypress/support/utils.ts

```diff
@@ -48,32 +86,42 @@ export function installChainlit() {
 }
 
 export function runSpec(test: string) {
-  return runCommand(`npx cypress run --spec cypress/e2e/${test}/spec.cy.ts`);
+  // const recordOptions = ` --record --key ${process.env.CYPRESS_RECORD_KEY} `;
```

Comment:
We could add a conditional, depending on whether the `process.env.CYPRESS_RECORD_KEY` env variable is set?
That would avoid the need for removing the comment + commit + push to run the CI again when debugging. And that would allow the PR runs to run correctly without needed the variable.

Reply:
I agree but this is a secret. It is annoying to add / remove the secret on the github UI before running the CI and could impact other runs

Reply:
Given we're using the `pull_request` github action, the secrets won't be applied to PR CI runs, right?
I don't have a strong opinion about this.